src/traversal/__tests__/writeRelayUpdatePayload-test.js

```diff
@@ -735,73 +734,6 @@ describe('writePayload()', () => {
       edgeID = generateClientEdgeID(connectionID, commentID);
     });
 
-    it('warns if the created `edge` field is missing in the payload', () => {
```

Comment:
maybe keep this test case and just remove the expectation about the warning? would be nice to test that the other parts still work.